packages/client-core/src/admin/components/Project/ProjectDrawer.tsx

```diff
@@ -69,15 +69,20 @@ const ProjectDrawer = ({ open, inputProject, existingProject = false, onClose, c
         }
 
   const projectUpdateStatus = useHookstate(getMutableState(ProjectUpdateState)[project.name]).value
+  const projectMutation = useMutation('project')
 
   const handleSubmit = async () => {
     try {
       if (existingProject && changeDestination) {
-        if (inputProject) await ProjectService.setRepositoryPath(inputProject.id, projectUpdateStatus.destinationURL)
```

Comment:
let's revert the ProjectService helpers, it's easier to read the code when these are self documenting functions